core/ui/EditTemplate/title.tid

```diff
@@ -35,10 +35,22 @@ tags: $:/tags/EditTemplate
 
 <$checkbox tiddler="$:/config/RelinkOnRename" field="text" checked="yes" unchecked="no" default="no"> {{$:/language/EditTemplate/Title/Relink/Prompt}}</$checkbox>
 
+<$reveal type="nomatch" state="$:/state/showeditreferencing" text="show">
```

Comment:
the state name could be more readable if it would be: `$:/state/show/references` or something similar. .. but I'm pretty sure we'll need to use `<<qualify..` .... 

Did you test the code if 2 tiddlers are open at the same time?

Reply:
As for the path of the state tiddler, you are right to be easier to read, but based on the `$:/state/showitpreview`, I would like to name it like it can be easily found. Anyway, I've modified it to `$:/state/edit/references`. I also applied the `qualify` macro.